.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Snuba Checks
+on:
+  push:
+    branches:
+      - master
+      - releases/**
+  pull_request:
+
+jobs:
+  tests:
+    name: Tests and code coverage
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    env:
+      CODECOV_TOKEN: 5338c540-6be8-4c46-beb0-bad8a1eb71bf
```

Comment:
This should probably be a repo secret

Reply:
Reset. It might actually be unnecessary from what I read.